person.rb

```diff
@@ -1,6 +1,6 @@
 # Defines a person class
```

Comment:
- Please make sure that your Person class inherits from Nameable: something like this `class Person < Nameable`
- Also, make sure that this class has a method `correct_name implemented`:
- Please these are the project requirements that you need to follow carefully. [Open here to learn more about Decorator pattern in Ruby ](https://refactoring.guru/design-patterns/decorator/ruby/example#example-0)

Reply:
In this PR
- [ ] Person class inherits from Nameable
- [ ] Person class has a method correct_name implemented: